class/systems.class.php

```diff
@@ -190,10 +190,37 @@ public function getSystemActivities($system)
 		$this->db->bind(':system', $system);
 		$result = $this->db->resultset();
 		$this->db->closeQuery();
-		
+
 		return $result;
 	}
 
+		/**
+	 * Get participants of currently active cache
+	 * @param unknown $system
+	 * @return string
+	 */
+	public function getActiveCacheParticipants($system)
+	{
+			$queryString = "SELECT Distinct Pilot
+					FROM activity
+					WHERE System = :system
+					AND ActivityDate >= (SELECT ActivityDate
```

Comment:
Some DB systems execute the inner query for each (!) selected row from the outer query. This can be a performance hit. Easier to read if you select the sowing activity date first in a separate query call and add this date later to the final query.